XML/TestCases/FunctionalTests-GPU.xml

```diff
@@ -3,7 +3,7 @@
         <testName>NVIDIA-CUDA-DRIVER-VALIDATION</testName>
         <testScript>GPU-DRIVER-INSTALL.ps1</testScript>
         <setupType>OneVM</setupType>
-        <OverrideVMSize>Standard_NC6</OverrideVMSize>
+        <OverrideVMSize>Standard_NC6s_v2</OverrideVMSize>
```

Comment:
This is not a simple VM size change. The GPU device under test changes from K80 to P100. But I think it is ok because P100 is later HW.